XRPLib/differential_drive.py

```diff
@@ -220,70 +219,49 @@ def turn(self, turn_degrees: float, max_effort: float = 0.5, timeout: float = No
         :type max_effort: float
         :param timeout: The amount of time before the robot stops trying to turn and continues to the next step (In Seconds)
         :type timeout: float
-        :param main_controller: The main controller, for handling the angle turned
-        :type main_controller: Controller
-        :param secondary_controller: The secondary controller, for maintaining position during the turn by controlling the encoder count difference
-        :type secondary_controller: Controller
-        :param use_imu: A boolean flag that changes if the main controller bases its movement off of the imu (True) or the encoders (False)
-        :type use_imu: bool
+        :param controller: The main controller, for handling the angle turned
+        :type controller: Controller
         :return: if the distance was reached before the timeout
         :rtype: bool
         """
-
+        
+        turn_speed = max_effort * 90
+        
         if max_effort < 0:
-            max_effort = -max_effort
             turn_degrees = -turn_degrees
```

Comment:
It feels like there should be a check for -degrees. What if I request -90 degrees at -1 speed?